speech/cloud-client/beta_snippets.py

```diff
@@ -126,6 +127,36 @@ def transcribe_file_with_auto_punctuation(path):
 # [END speech_transcribe_file_with_auto_punctuation]
 
 
+# [START speech_transcribe_diarization]
+def transcribe_file_with_diarization(path):
+    """Transcribe the given audio file synchronously with diarization."""
+    client = speech.SpeechClient()
+
+    with open(path, 'rb') as audio_file:
+        content = audio_file.read()
+
+    audio = speech.types.RecognitionAudio(content=content)
+
+    config = speech.types.RecognitionConfig(
+        encoding=speech.enums.RecognitionConfig.AudioEncoding.LINEAR16,
+        sample_rate_hertz=16000,
+        language_code='en-US',
+        enable_speaker_diarization=True,
+        diarization_speaker_count=2)
+
+    print('Waiting for operation to complete...')
+    response = client.recognize(config, audio)
+
+    for i, result in enumerate(response.results):
+        alternative = result.alternatives[0]
+        print('-' * 20)
+        print('First alternative of result {}: {}'
+              .format(i, alternative.transcript))
+        print('Speaker Tag for the first word: {}'
+              .format(alternative.words[0].speaker_tag))
```

Comment:
does each `word` in `alternative.words` have other relevant information concerning diarization besides `speaker_tag`?  if so print them as well, if not please ignore this comment.